src/contracts/core/RewardsCoordinator.sol

```diff
@@ -488,8 +505,13 @@ contract RewardsCoordinator is
         return _distributionRoots[index];
     }
 
+    /// @notice loop through the distribution roots from reverse and get latest root that is not disabled
     function getCurrentDistributionRoot() external view returns (DistributionRoot memory) {
-        return _distributionRoots[_distributionRoots.length - 1];
+        for (uint256 i = _distributionRoots.length; i > 0; i--) {
+            if (!_distributionRoots[i - 1].disabled) {
+                return _distributionRoots[i - 1];
+            }
+        }
```

Comment:
Fairly unrelated to this PR, but is the intention here that this is the latest *active* root, or the latest root in general (which may be waiting to become active)?
Seems like this is written to be the latter; I think that's an OK choice but we should be explicit about the intention.

Reply:
Yeah wasn't actually sure on the desired behavior from the view funcs here. Perhaps multiple interfaces could be added including one that finds the latest root that is both active AND not disabled

Reply:
I'd be fine with both, personally. Could be good to ask a question or two (on Slack) + consider naming for these function(s) to maximize clarity. I don't think this PR needs to be the place to make this decision though.